### PR TITLE
Feature/renew build

### PR DESCRIPTION
**Pull request by takama**

### CODE REVIEW

File: roles/build/tasks/main.yml [hunk 1]
```diff
@@ -49,6 +49,17 @@
     src: kubeconfig.j2
     dest: "{{ ansible_env.HOME }}/.kube/config"
 
+- name: Deploy script for cockroach databases/users
```
Comment (STASiAN) [inline roles/build/tasks/main.yml:52]: why cockroach deploy is here: roles/build/tasks/main.yml ???

File: roles/build/templates/deploy-k8s-community-user-db.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```
Comment (STASiAN) [inline roles/build/templates/deploy-k8s-community-user-db.sh:1]: why you place deploy-k8s-community-user-db.sh in build role?

File: roles/build/tasks/main.yml (hunk 1 above)
Comment (takama) [reply roles/build/tasks/main.yml:52]: This is renew step only (due to time limit) it contains another data related with cicd.
Next step will be as separate roles for build machine and for cicd/database settings

File: roles/build/templates/deploy-k8s-community-user-db.sh (hunk 2 above)
Comment (takama) [reply roles/build/templates/deploy-k8s-community-user-db.sh:1]: The same use-case described above. Right now I placed nothing (5% dependency) of already depended role.

**Review by STASiAN — APPROVED** [review]

**Pull request merged**